.github/workflows/gpu.yml

```diff
@@ -100,3 +104,71 @@ jobs:
       - name: Run benchmarks
         working-directory: ${{env.BUILD_DIR}}
         run: ctest --output-on-failure --test-dir benchmark -C ${{env.BUILD_TYPE}} --exclude-regex umf-bench-multithreaded
+
+  gpu-CUDA:
+    name: Build
+    env:
+      BUILD_TYPE: Release
+    # run only on upstream; forks will not have the HW
+    # if: github.repository == 'oneapi-src/unified-memory-framework'
```

Comment:
I suppose this should be uncommented?

Reply:
only when we got CUDA machines in our CI